js/apps/admin-ui/src/client-scopes/details/ScopeForm.tsx

```diff
@@ -37,8 +37,8 @@ type ScopeFormProps = {
 };
 
 export const ScopeForm = ({ clientScope, save }: ScopeFormProps) => {
-  const { t } = useTranslation("client-scopes");
-  const { t: tc } = useTranslation("clients");
+  const { t } = useTranslation();
+  const { t: tc } = useTranslation();
```

Comment:
We can remove this line here, as we only need one instance of `t`. You can probably search the project for `t: ` to find other instances where we are aliasing this.